app/src/main/java/com/col/eventradar/ui/bottom_sheets/EventDetailsBottomSheet.kt

```diff
@@ -0,0 +1,60 @@
+package com.col.eventradar.ui.bottom_sheets
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import com.col.eventradar.databinding.FragmentEventDetailsBottmSheetBinding
+import com.google.android.material.bottomsheet.BottomSheetDialogFragment
+
+data class EventDetails(
```

Comment:
```suggestion
data class EventDetails(
    val type: EventType,
    val name: String,
    val locationName: String,
    val time: String,
    val description: String,
    val commentsAmount: Int,
)
```
`type` should be an enum

Reply:
👍